crates/quickjs-wasm-rs/src/js_binding/context.rs

```diff
@@ -141,6 +141,24 @@ impl JSContextRef {
         JSValueRef::new(self, raw)
     }
 
+    /// Returns the raw pointer to the underlying [quickjs_wasm_sys::JSContext].
+    ///
+    /// # Safety
+    /// While calling this function is safe, using it’s return value has to be
+    /// done with a sufficitenly deep understanding of QuickJS and [quickjs_wasm_sys].
+    pub unsafe fn as_raw(&self) -> *mut JSContext {
+        self.inner
+    }
+
+    /// Creates a `JSContextRef` from a pointer to a [quickjs_wasm_sys::JSContext].
+    ///
+    /// # Safety
+    /// The caller has to ensure that the returned `JSContextRef` is the only one
+    /// used throughout its lifetime.
+    pub unsafe fn from_raw(inner: *mut JSContext) -> JSContextRef {
+        JSContextRef { inner }
+    }
+
```

Comment:
Could you add a comment to each, stating that given the safety implications of these methods, we're not making any commitments on maintaining them and that they can be removed in the future?